src/planning/path_planner/include/path_planner/map_utils.hpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Package:   path_planner
+ * Filename:  map_utils.hpp
+ * Author:    Jim Moore
+ * Email:     jim3moore@gmail.com
+ * Copyright: 2021, Nova UTD
+ * License:   MIT License
+ */
+
+#ifndef PATH_PLANNER_MAP_UTILS_HPP
+#define PATH_PLANNER_MAP_UTILS_HPP
+
+
+#include "autoware_auto_msgs/msg/trajectory.hpp"
+#include <lanelet2_core/geometry/Lanelet.h>
+#include <lanelet2_core/LaneletMap.h>
+#include <lanelet2_core/primitives/Point.h>
+#include <lanelet2_core/geometry/LineString.h>
+#include <autoware_auto_msgs/msg/trajectory.hpp>
```

Comment:
We should transition to using `voltron_msgs` in the future, migrating and modifying Autoware msg files as necessary. This gives us greater clarity and control over our message formats. And I suspect that Autoware's message approach can be simplified for our uses...

Reply:
Sounds good